### PR TITLE
format.ps1xml generator upgraded to .Net Standard

### PR DESCRIPTION
**Pull request by vladimir-shcherbakov**
## Description

Issue ##8309

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
@vladimir-shcherbakov Please reassign to someone else. Thanks.

File: documentation/development-docs/generate-format.ps1xml-file.md [hunk 1]
```diff
@@ -180,18 +180,18 @@ namespace Microsoft.Azure.Commands.Profile.Models
 
 // code omitted for brevity
 
-        [Ps1Xml(Label = "Subscription Id", Target = ViewControl.Table)]
+        [Ps1Xml(Label = "SubscriptionId", Target = ViewControl.Table)]
```
Comment (markcowl) [inline documentation/development-docs/generate-format.ps1xml-file.md]: Why are the spaces being removed from the column headings?

File: documentation/development-docs/generate-format.ps1xml-file.md [hunk 2]
```diff
@@ -396,7 +413,7 @@ PS C:\> Import-Module E:\git\azure-powershell\artifacts\Debug\Az.Accounts\Az.Acc
 ```Powershell
 PS C:\> Get-AzSubscription
 
-Subscription Id                      Subscription Name         State   Tenant Id
+SubscriptionId                       SubscriptionName          State   TenantId
```
Comment (markcowl) [inline documentation/development-docs/generate-format.ps1xml-file.md]: Same question

File: tools/FormatPs1XmlGenerator/FormatPs1XmlGenerator/FormatPs1XmlGenerator.psd1 [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+PowerShellVersion = '3.0'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module
+DotNetFrameworkVersion = '4.5.2'
```
Comment (markcowl) [inline tools/FormatPs1XmlGenerator/FormatPs1XmlGenerator/FormatPs1XmlGenerator.psd1]: 4.7.2

File: tools/FormatPs1XmlGenerator/FormatPs1XmlGenerator/FormatPs1XmlGenerator.psd1 [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Generates format.ps1xml files for a provided module'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '3.0'
```
Comment (markcowl) [inline tools/FormatPs1XmlGenerator/FormatPs1XmlGenerator/FormatPs1XmlGenerator.psd1]: 5.1

File: documentation/development-docs/generate-format.ps1xml-file.md (hunk 1 above)
Comment (vladimir-shcherbakov) [reply documentation/development-docs/generate-format.ps1xml-file.md]: It's recommended to give a real property name so customer may use it for a next query, for example
```
PS C:\> Get-AzSubscription

SubscriptionId                       SubscriptionName          State   TenantId
---------------                      -----------------         -----   ---------
c9cbd920-c00c-427c-852b-c329e824c3a8 Azure SDK Powershell Test Enabled 72f988bf-86f1-41af-91ab-7a64d1d63df5
6b085460-5f21-477e-ba44-4cd9fbd030ef Azure SDK Infrastructure  Enabled 72f988bf-86f1-41af-91ab-7a64d1d63df5

PS C:\> Get-AzSubscription | ? {$_.SubscriptionName  -like "*Test*" }
```

**Pull request merged**